evm_loader/program/src/account_storage/apply.rs

```diff
@@ -267,6 +278,63 @@ impl<'a> ProgramAccountStorage<'a> {
         Ok(())
     }
 
+    fn apply_withdrawals(&mut self,

```

Comment:
You forgot to actually change user balance in EthereumAccount struct

Reply:
Fixed